README.md

```diff
@@ -39,6 +40,12 @@ Clone the collection repository.
   git clone https://github.com/redhat-cop/cloud.gcp_ops .
 ```
 
+Install the python requirements.
+
+```shell
+  pip install -r requirements.txt
```

Comment:
I'm not really sure if we should include in this collection the requirements of the google.cloud collection. In the README we already mention that the google.cloud collection must be installed before using this one. We should not need to add a requirements.txt file in my opinion.